test/files/pos/t2377b/a.scala

```diff
@@ -0,0 +1,3 @@
+object Test {
+  (new Q.Inner).foo // make sure foo's type is well-formed
```

Comment:
It would strenghten the test to have: `.foo.someMethodOnInner_Builder` to check the right type is resolved.